### DISCUSSION TITLE
How to change icon in open/closed Accordion programmatically ?

### DISCUSSION THREAD

**Discussion by vaibhavsingh97** [radix-ui/primitives#1804] [Help] [answered]
Hey 👋,

I'm asking for help changing the icon of the `open/closed` state of the accordion programmatically.  I looked through the documentation and found no answers to this question.

The only thing mentioned in the doc is around, how to [rotate the icon when the accordion is open ](https://www.radix-ui.com/docs/primitives/components/accordion#rotated-icon-when-open) which is using CSS

Any ideas and workaround would be appreciated.

Thank you

**Comment by joaom00** [comment, marked as answer]
Hey @vaibhavsingh97,

You can control the `open/closed` state with `value` and `onValueChange` props

**Reply by vaibhavsingh97** [reply]
Thanks @joaom00 !! It worked
